conslogging/conslogging.go

```diff
@@ -256,7 +255,7 @@ func (cl ConsoleLogger) PrintBytes(data []byte) {
 	}
 	if len(output) > 0 {
 		c.Fprintf(cl.outW, "%s", string(output))
-		output = output[:0]
+		// output = output[:0] // not needed
```

Comment:
Nit: if its not needed we should delete it

Reply:
I'd leave it in case we need to add more code after that line that uses output somehow. Then it is needed and it might not be obvious.

Reply:
Nit on the nit: Maybe note in the comment when it would be needed since its not obvious